Formula/jpeg-xl.rb

```diff
@@ -37,11 +39,6 @@ class JpegXl < Formula
 
   # These resources are versioned according to the script supplied with jpeg-xl to download the dependencies:
   # https://github.com/libjxl/libjxl/tree/v#{version}/third_party
-  resource "highway" do
-    url "https://github.com/google/highway.git",
-        revision: "22e3d7276f4157d4a47586ba9fd91dd6303f441a"
```

Comment:
This is actually the commit has of the release tag, so making this change seems pretty safe.

https://github.com/google/highway/releases/tag/1.0.1